docs/customizations/prechatsurveypane.md

```diff
@@ -86,15 +88,20 @@ liveChatWidgetProps = {
 };
 ...
 ```
+
 </details>
 
 <img src="../.attachments/customizations-prechat-adaptive-card-host-config-change-background-color.png" width="450">
 
 ----------
 
 ### Changing adaptive card payload
+
+__NOTE__ : The goal of this sample, is to demonstrate HOW TO pass the props value to override the pre-chat survey, It is mandatory to match the format of the Adaptive Card with the same elements used by the Out-of-the-Box widget, which this sample may not fully cover. Specifically, this includes the format for the `Id` object.

```

Comment:
It would be nice if there was a link we can give the reader for the adaptive card formats

Reply:
We dont really have formats, and we dont control changes on ADC for prechat,